core/src/test/java/org/infinispan/expiration/impl/ClusterExpirationFunctionalTest.java

```diff
@@ -340,4 +341,49 @@ public void testWriteExpiredEntry() throws InterruptedException {
          ts2.advance(secondOneMilliAdvanced);
       }
    }
+
+   public void testMaxIdleReadNodeDiesPrimary() {
+      testMaxIdleNodeDies(c -> {
+         if (cacheMode.isScattered()) {
```

Comment:
`MultipleCacheManagersTest#getKeyForCache(org.infinispan.Cache<?,?>, org.infinispan.Cache<?,?>...)`

Reply:
Ah nice, wasn't aware of that.